### PR TITLE
Feature/bookmarks fetch

### PR DESCRIPTION
**Pull request by Barylo**
Guys, I need your help with review my code. For now I see this screen on BookmarksList page.
image

May be something wrong with code in BookmarksList component here:
image

I would appreciate your collaboration!

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
CI failed, please check

**Review by DavidKorystov — APPROVED** [review]

File: src/types/index.ts [hunk 1]
```diff
@@ -87,3 +87,10 @@ export interface User extends MongoArtifacts {
     };
   };
 }
+
+export interface Bookmark extends Document {
```
Comment (ArtemLigerko) [inline src/types/index.ts]: export interface Bookmark extends MongoArtifacts {

File: src/containers/BookmarksList.tsx [hunk 2]
```diff
@@ -63,21 +94,19 @@ const BookmarksList: React.FC = () => {
...
-          likeCount={item.likeCount}
-        />
-      ))}
+      <>
+        {bookmarks.docs.length ? (
+          <InfiniteList<Bookmark>
+            showMore={handleShowMore}
+            data={bookmarks}
+            itemComponent={(itemData) => (
+              <FeedSingleTweet {...itemData.tweet} {...itemData.owner} />
```
Comment (ArtemLigerko) [inline src/containers/BookmarksList.tsx]: add key={itemData._id} to < FeedSingleTweet />

**Comment by ArtemLigerko** [comment]
Good working! And make, please, some fix:
1. Add: 
```
const user = useSelector<RootState, RootState["user"]>((store) => store.user);
```
2. Replace: @artlee4 in 77 line with 
```
{user._id ? `@${user.username}` : ""}

```

did it, but bookmarks still do not fetch

**Comment by ArtemLigerko** [comment]
> 
WM-Screenshots-20220922120842
strange, it is working for me, maby here is another problem

**Review by ArtemLigerko — APPROVED** [review]

**Pull request merged**